addon/_locales/zh_TW/long_description.html

```diff
@@ -0,0 +1,9 @@
+當你點選單按鈕時，這個套件會<strong>偵測</strong>目前頁面的開放街圖參數，並且會<strong>產生</strong>相關地圖與對應參數清單。
+
+認可的參數包括:
+<ul>
+<li>地圖經緯度 (縮放、經度、緯度)</li>
+<li>OSM 元素 (節點 id、路徑 id、關聯 id 與變更組合 id)</li>
+<li>使用者名稱與 GPS 軌跡 id</li>
+</ul>
+如果你不確定那些網站能被認可，在點選單之前請瀏覽 https://www.openstreetmap.org/ 網站。
```

Comment:
Looks like there is a missing line:
> <em>The OpenStreetMap logo is a trademark of the OpenStreetMap Foundation, and is used with their permission. This project is not endorsed by or affiliated with the OpenStreetMap Foundation.</em>

Could you translate it also? :-)

Reply:
I will have to add this line in english for now, but feel free to translate it when you can.